frontend/src/app/Settings/Users/ManageUsers.tsx

```diff
@@ -0,0 +1,201 @@
+import UserFacilitiesSettingsForm from "./UserFacilitiesSettingsForm";
+import UserRoleSettingsForm from "./UserRoleSettingsForm";
+import { SettingsUser, UserFacilitySetting } from "./ManageUsersContainer";
+import { showNotification } from "../../utils";
+
+import "./ManageUsers.scss";
+
+interface Props {
+  loggedInUser: User;
+  users: SettingsUser[];
+  allFacilities: UserFacilitySetting[];
+  onUpdateUser: (user: SettingsUser) => void;
+}
+
+type SettingsUsers = { [id: string]: SettingsUser };
+
+const ManageUsers: React.FC<Props> = ({
+  allFacilities,
+  loggedInUser,
+  users,
+  onUpdateUser,
+}) => {
+  let settingsUsers: SettingsUsers = users.reduce(
+    (acc: SettingsUsers, user: SettingsUser) => {
+      acc[user.id] = user;
+      return acc;
+    },
+    {}
+  );
+
+  const [usersState, updateUsersState] = useState<SettingsUsers>(settingsUsers);
+  const [activeUserId, updateActiveUserId] = useState<string>(
+    Object.keys(settingsUsers)[0]
+  ); // TODO: unless there is a desired sort algorithm, pick the first user
+  const [nextActiveUserId, updateNextActiveUserId] = useState<string | null>(
+    null
+  );
+  const [showInProgressModal, updateShowInProgressModal] = useState(false);
+
+  function updateUser<T>(
+    userId: string,
+    key: string, // the field to update
+    value: T // value of the field to update
+  ) {
+    updateUsersState({
+      ...usersState,
+      [userId]: {
+        ...usersState[userId],
+        [key]: value,
+        isEdited: true,
+      },
+    });
+  }
+
+  const onSaveChanges = (userId: string) => {
+    onUpdateUser(usersState[userId]); // TODO this does nothing atm
+
+    updateUsersState({
+      ...usersState,
+      [userId]: {
+        ...usersState[userId],
+        isEdited: false,
+      },
+    });
+
+    let successAlert = (
+      <Alert
+        type="success"
+        title="Changes Saved"
+        body={`${usersState[userId].name}'s settings have been saved`}
+      />
+    );
+
+    showNotification(toast, successAlert);
+  };
+
+  const onChangeActiveUser = (nextActiveUserId: string) => {
+    if (usersState[activeUserId].isEdited) {
+      updateNextActiveUserId(nextActiveUserId);
+      updateShowInProgressModal(true);
+    } else {
+      updateActiveUserId(nextActiveUserId);
+    }
+  };
+
+  const onContinueChangeActiveUser = (currentActiveUserId: string) => {
+    updateShowInProgressModal(false);
+    updateActiveUserId(nextActiveUserId as string);
+    resetUser(currentActiveUserId);
+  };
+
+  const sideNavItems = Object.values(usersState).map((user: any) => {
+    return (
+      <li
+        className="usa-sidenav__item users-sidenav-item"
+        onClick={() => onChangeActiveUser(user.id)}
+        key={user.id}
+      >
+        <ConditionalWrap
+          condition={activeUserId === user.id}
+          wrap={(children) => <div className="usa-current">{children}</div>}
+        >
+          <div className="padding-105 padding-right-2">
```

Comment:
Instead of using a `ConditionalWrap` component you could just add the `usa-current` class to this one. The `classname` module will make that easy: 
```
<div className={
   classname("padding-105 padding-right-2", 
   activeUserId === user.id && "usa-current")
}>
```